recipe/meta.yaml

```diff
@@ -9,37 +9,37 @@ package:
 source:
   fn: {{ version }}.tar.gz
   url: https://github.com/santiagoropb/pynumero_libraries/archive/v{{ version }}.tar.gz
-  sha256: cf21b5045aa18c636b4e2e13fe4f42119b0a4b0ccef96e43d4e5876f675d78a7
+  sha256: d08985d7832e30db6d45998399178cd3d4b96db6931ec400107066a354a7f066
     
 build:
   number: 0
-  skip: true  # [win]
-      
+  skip: true  # [win and vc<14]
+  
 requirements:
   build:
     - cmake
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - ampl-mp
-    - python
-    - numpy
-    - pymumps
-    - matplotlib
-    - scipy
+    - ampl-mp  # [not win]
+    - python  # [not win]
```

Comment:
I am bit confused here. Why don't you need Python on Windows?

Reply:
I do need it. I just remove it because I wanted to see the cmake project to build but I was planning on adding it again

Reply:
Let me clean up the recipe with your suggestions to get it working

Reply:
Sure. Nevermind :)

Reply:
Thanks a lot for the help. I really appreciate it!

Reply:
After the changes it is not finding the .lib file. Any idea how to fix that problem?

Reply:
No, sorry, I don't really know how cmake works. I mean, I see `TARGET_LINK_LIBRARIES( asl_test pynumero_ASL)` but how would cmake know where that library lives since it has not been installed yet?

Reply:
If I skip the test the libraries do not get package?

Reply:
It should not affect the packaging of the libraries.

Reply:
But it's probably a good idea  to use this as a test that you can actually link something against them.